packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -89,25 +84,18 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 
 	const reset = async () => provider.reset();
 
-	const allDataCorruption = async (containers: IContainer[]) =>
-		Promise.all(
-			containers.map(
-				async (c) =>
-					new Promise<boolean>((resolve) =>
-						c.once("closed", (error) => {
-							resolve(error?.errorType === ContainerErrorType.dataCorruptionError);
-						}),
-					),
-			),
-		).then((all) => !all.includes(false));
-
 	const runtimeOf = (dataObject: ITestFluidObject): IContainerRuntime =>
 		dataObject.context.containerRuntime as IContainerRuntime;
 
 	const createDataStoreWithProps = async (dataObject: ITestFluidObject, id: string) =>
 		runtimeOf(dataObject)._createDataStoreWithProps(packageName, {}, id);
 
-	const getRootDataStore = async (dataObject: ITestFluidObject, id: string, wait = true) =>
+	const getAliasedDataStore = async (dataObject: ITestFluidObject, id: string, wait = false) =>
```

Comment:
Pretty sure you can remove the `wait` param and hardcode it to false when calling `getRootDataStore` (I don't see anyone passing true)